app/src/main/java/com/amaze/filemanager/activities/TextEditorActivity.java

```diff
@@ -285,7 +285,8 @@ private void saveFile(final String editTextString) {
      * on a worker thread
      */
     private void load() {
-        mInput.setHint(R.string.loading);
+        Toast.makeText(getApplicationContext(), R.string.loading, Toast.LENGTH_SHORT).show();
```

Comment:
I think this should be a snackbar. For error we are not using snackbars because the activity is destroyed (and undissmissable dialogs are undissmissable).